web/src/tippyjs.js

```diff
@@ -262,25 +300,16 @@ export function initialize() {
     delegate("body", {
         target: ".recipient_bar_icon",
         onShow(instance) {
-            if (!document.body.contains(instance.reference)) {
-                return false;
+            // Don't push the instance to the tippyInstances array if it already exists.
+            if (!tippyInstances.includes(instance)) {
+                tippyInstances.push(instance);
             }
-            const $elem = $(instance.reference);
-
-            const config = {attributes: false, childList: true, subtree: true};
-            const target = $elem.parents(".message_header.message_header_stream.right_part").get(0);
-            const nodes_to_check_for_removal = [
-                $elem.parents(".recipient_bar_controls").get(0),
-                $elem.get(0),
-            ];
-            hide_tooltip_if_reference_removed(target, config, instance, nodes_to_check_for_removal);
-            return true;
         },
         onHidden(instance) {
+            // we need to remove the instance from the tippyInstances array since the
+            // reference is removed from the DOM.
+            tippyInstances = tippyInstances.filter((i) => i !== instance);
             instance.destroy();
-            if (observer) {
-                observer.disconnect();
-            }
```

Comment:
Similarly, this should not be required as well.

Reply:
Removed it.